api/requests.go

```diff
@@ -147,18 +136,12 @@ func (a *API) AddRequestHandler(ctx echo.Context) error {
 		return err
 	}
 
-	// Format a human readable copy of the request submission details.
-	humanReadableRequestDetails, err := formatRequestDetails(requestSubmission.Details.(map[string]interface{}))
-	if err != nil {
-		return err
-	}
-
 	// Add required information to a copy of the request details.
 	requestDetails := copyRequestDetails(requestSubmission.Details.(map[string]interface{}))
 	requestDetails["username"] = user
 	requestDetails["request_id"] = requestID
 	requestDetails["request_type"] = requestType.Name
-	requestDetails["request_details"] = humanReadableRequestDetails
+	requestDetails["request_details"] = requestSubmission.Details.(map[string]interface{})
```

Comment:
This information is now redundant, but I thought I should leave it in place in case something is still looking for it.